manual/manual/library/libunix.etex

```diff
@@ -39,20 +38,18 @@ more information on the functions that are not supported under Windows.
 \end{windows}
 
 \begin{latexonly}
-{
-\ocamldocinputstart
-\input{Unix.tex}
+\begin{linklist}
+\stddocitem{Unix}{Unix system calls}
 
-\section{s:Module \texttt{UnixLabels}: labelized version of the interface}
+\section{Module \texttt{UnixLabels}: labelized version of the interface}
```

Comment:
the `s:` looks like a search-and-replace bug here, right? But how come we are not using the new `\section{label}{title}` macro?

Reply:
I don't remember how I inserted this `s:` here.

Part of the issue is that this section heading is the only one in the manual that is inside an ocamldocinput section, which reinstall the normal section macro.

I have pushed a change to get out earlier of the ocamldocinput mode and avoid this corner case.

Reply:
In the new patch, the label is `Unixlabels` when I would maybe have expected `s:Unixlabels`, but I trust you to have done the right thing.

Reply:
I have merely kept the previous label since the label is unused, not in conflict with other labels, and only exists in the pure latex version of the manual.